common/presenters/population-to-grid.js

```diff
@@ -0,0 +1,34 @@
+function populationToGrid({ population } = {}) {
+  if (population === undefined) {
+    return {}
+  }
+
+  return {
+    details: {
+      date: population.date,
```

Comment:
Does date actually need to come from here? I'm guessing (in theory) this should match the date from the `req.params` because it was returned as part of the search on that date. 

So maybe for consistency and simplicity the view just uses the one variable?

Reply:
We only have the date returned from the populations endpoint, if there was population data. Otherwise we have to use the date as provided as the `req.params`.

I shouldn't make any different, but I was coding defensively and using the known data from the backend in the situation where it was available.

@teneightfive  If you're prefer to only use `req.params.date` I can change it to use that everywhere we're displaying a date.

Reply:
Changed this to only use the `req.params.date`

Reply:
Yeah I think it feels cleaner to use the date that the user is in theory viewing (which is set in the URL) for both instances.

I guess the worry is that the API returns a different date as part of the population, but even doing it this way, we wouldn't really _know_.